torch/nn/modules/__init__.py

```diff
@@ -27,6 +27,8 @@
 from .distance import PairwiseDistance, CosineSimilarity
 from .fold import Fold, Unfold
 from .adaptive import AdaptiveLogSoftmaxWithLoss
+from .transformer import TransformerBase, TransformerEncoder, TransformerDecoder, \
```

Comment:
Why are the modules other than `Transformer` exposed?

Reply:
sometime users may use parts of the transformer model, so the encoder/decoder  modules are exposed. I agree TransformerBase should not be exposed.